README.md

```diff
@@ -29,6 +29,14 @@
 
 _Formerly know as `bpfd`_
 
+bpfman is a Cloud Native Computing Foundation Sandbox project
+
+<picture>
+   <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/cncf/artwork/main/other/cncf/horizontal/white/cncf-white.png"/>
+   <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/cncf/artwork/main/other/cncf/horizontal/color/cncf-color.png"/>
+   <img alt="CNCF Logo" src="https://raw.githubusercontent.com/cncf/artwork/main/other/cncf/horizontal/color/cncf-color.png" width="200px"/>
+</picture>
```

Comment:
It might be how it was rendored, but I previewed this in Chrome in Dark Mode and it didn't switch images.